plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -8,15 +8,23 @@
             description="%runtimeTypeDescription"
             id="com.google.cloud.tools.eclipse.appengine.standard.runtime"
             name="%apptoolsRuntimeTypeName"
+            supportsManualCreation="false"
             vendor="%runtimeTypeVendor"
             version="1">
+         <!-- J2EE Web Modules -->
          <moduleType
                types="jst.web"
                versions="2.5">
          </moduleType>
+         <!-- Static Web Projects -->
```

Comment:
This addition is good. Would it make sense to pull it out as a separate, quicker PR?